index.js

```diff
@@ -56,14 +58,34 @@ function groups (groups, data, aggregations, thisArg, additionalArgs) {
   data = Array.isArray(data) ? data : data.features
   var args = [undefined, undefined, undefined].concat(additionalArgs)
 
+  var tree = rbush(data.length)
```

Comment:
^ why `data.length` as the branching factor?  intuitively I would think this would make the search suboptimally slow.